Utilities/Scan-AllStorageAccountsAccessLevel.ps1

```diff
@@ -0,0 +1,138 @@
+    exit 1
+}
+
+#region HTML File structure
+
+$TableStyle = '
+<style type="text/css">
+  .tm  {border-collapse:collapse;border-spacing:0;border-color:#999;}
+  .tm td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#999;color:#444;background-color:#F7FDFA;}
+  .tm th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#999;color:#fff;background-color:#26ADE4;}
+  .tm .tm-dk6e{font-weight:bold;color:#ffffff;text-align:center;vertical-align:top}
+  .tm .tm-xa7z{background-color:#ffccc9;vertical-align:top}
+  .tm .tm-ys9u{background-color:#b3ffd9;vertical-align:top}
+  .tm .tm-7k3a{background-color:#D2E4FC;font-weight:bold;text-align:center;vertical-align:top}
+  .tm .tm-yw4l{vertical-align:top}
+  .tm .tm-6k2t{background-color:#D2E4FC;vertical-align:top}
+</style>
+'
+
+$htmlHeader = '
+<h2>&bull;&nbsp;SUBSCRIPTION_IDENTIFIER</h2>
+<table class="tm">
+  <tr>
+    <th class="tm-dk6e" colspan="9">Storage Accounts with Anonymous Access</th>
+  </tr>
+  <tr>
+    <td class="tm-7k3a">Sr</td>
+    <td class="tm-7k3a">Storage Account</td>
+    <td class="tm-7k3a">Container Name</td>
+    <td class="tm-7k3a">Public Access Level</td>
+    <td class="tm-7k3a">Location</td>
+  </tr>
+'
+
+$htmlNodeRed =
+'
+  <tr>
+    <td class="tm-yw4l">SR_ID</td>
+    <td class="tm-yw4l">STORAGE_ACCOUNT</td>
+    <td class="tm-yw4l">CONTAINER_NAME</td>
+    <td class="tm-yw4l">PUBLIC_ACCESS_LEVEL</td>
+    <td class="tm-yw4l">LOCATION</td>
+  </tr>
+'
+
+$htmlEnd =
+'
+</table>
+'
+#endregion
+
+$ReportHTMLFile = "AnonymousAccessResource.html"
+if (!(Test-Path -Path ".\AnonymousAccessResource.html")) {
+    $htmlHeader = $TableStyle + $htmlHeader
+}
+
+#region Get Subscription Details...
+Write-LogInfo "Running: Get-AzSubscription..."
+$subscription = Get-AzSubscription
+$htmlHeader = $htmlHeader.Replace("SUBSCRIPTION_IDENTIFIER","$($subscription.Name) [$($subscription.Id)]")
+#endregion
+
+#region Get all anonymous access containers and build HTML Page
+$EmailSubjectTextFile = ".\ShowAnonymousAccessResourceEmailSubject.txt"
+$finalHTMLString = $htmlHeader
+$StorageAccountHtml = '<a href="https://ms.portal.azure.com/#resource/subscriptions/' + "$($subscription.Id)" + '/resourceGroups/RESOURCE_GROUP_NAME/providers/Microsoft.Storage/storageAccounts/STORAGE_NAME/overview" target="_blank" rel="noopener">STORAGE_NAME</a>'
+$ContainerHtml = '<a href="https://ms.portal.azure.com/#blade/Microsoft_Azure_Storage/ContainerMenuBlade/overview/storageAccountId/%2Fsubscriptions%2F' + "$($subscription.Id)" + '%2FresourceGroups%2FRESOURCE_GROUP_NAME%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2FSTORAGE_NAME/path/CONTAINER_NAME" target="_blank" rel="noopener">CONTAINER_NAME</a>'
+
+$then = Get-Date
+Write-LogInfo "Elapsed Time: $($(Get-Date) - $then)"
```

Comment:
Please format for human-readable.
Your result is [INFO ] Elapsed Time: 00:00:00.0070034

Reply:
I have changed to below format.
Elapsed Time: 0 hours 2 minutes 58 seconds